surrealdb/ws.py

```diff
@@ -487,7 +465,7 @@ async def create(
     async def update(
         self, thing, data: Optional[Dict[str, Any]]
```

Comment:
Hi there! I noticed that `thing` is missing a type annotation. I don't know if mypy ignores this, but pyright on strict mode complains, "Type of 'update' is partially unknown." This is because `thing`'s data type isn't indicated